spec/support/features/session_helper.rb

```diff
@@ -158,8 +158,9 @@ def begin_sign_up_with_sp_and_ial(ial2:)
 
     def sign_up_and_set_password
       user = sign_up
-      fill_in t('forms.password'), with: VALID_PASSWORD
-      fill_in t('components.password_confirmation.confirm_label'), with: VALID_PASSWORD
+      user.password = VALID_PASSWORD
```

Comment:
Setting this here so that sign_in_live_with_2fa(user) is successful after signing out. This helper is used all over, so wondering if this might hide issues in the future.